backend/src/main/scala/cromwell/backend/WorkflowBackendActor.scala

```diff
@@ -75,4 +86,9 @@ trait WorkflowBackendActor extends Actor with ActorLogging {
     * Abort a running job.
     */
   def abort(jobKey: BackendJobDescriptorKey): Future[AbortResponse]
+
+  /**
+    * Validate that this WorkflowBackendActor can run all of the calls that it's been assigned
+    */
+  def validate(): Future[ValidationResponse]
```

Comment:
This should be blocking. In other words Unit. If there is a problem should throw an exception otherwise nothing.